builtins.go

```diff
@@ -464,7 +450,10 @@ func builtinSort(i *interpreter, trace traceElement, arguments []value) (value,
 		}
 	}
 
-	data.Sort()
+	err = data.Sort()
+	if err != nil {
```

Comment:
This is probably the most consequential part of this change. In particular one of our tests already covers this, but I have missed it before (it was committed with a bad golden).